scripts/lib/get_e3sm_tests.py

```diff
@@ -1,193 +1,10 @@
-                         "ERS_IOP4p.f19_g16_rx1.A",
-                         "ERS_IOP4p.ne30_g16_rx1.A",
-                         "HOMME_P24.f19_g16_rx1.A",
-                         "NCK.f19_g16_rx1.A",
-                         "SMS.ne30_f19_g16_rx1.A",
-                         "ERS_Ld5.T62_oQU120.CMPASO-NYF",
-                         "ERS.f09_g16_g.MPASLISIA",
-                         "SMS.T62_oQU120_ais20.MPAS_LISIO_TEST",
-                         "SMS.f09_g16_a.IGCLM45_MLI"
-                        ,("SMS_P12x2.ne4_oQU240.A_WCYCL1850","mach_mods")
-                        )),
-
-    "e3sm_integration" : (("e3sm_developer", "e3sm_atm_integration"),"03:00:00",
-                          ("ERS.ne11_oQU240.A_WCYCL1850",
-		           ("SMS_D_Ld1.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6","v1cmip6"),
-                           "ERS_Ln9.ne4_ne4.FC5AV1C-L",
-                          #"ERT_Ld31.ne16_g37.B1850C5",#add this line back in with the new correct compset
-                           ("PET.f19_g16.X","mach-pet"),
-                           ("PET.f45_g37_rx1.A","mach-pet"),
-                           ("PET_Ln9_PS.ne30_oECv3_ICG.A_WCYCL1850S","mach-pet"),
-                           "PEM_Ln9.ne30_oECv3_ICG.A_WCYCL1850S",
-                           "ERP_Ld3.ne30_oECv3_ICG.A_WCYCL1850S",
-                           "SEQ_IOP.f19_g16.X",
-                           "SMS.f09_g16_a.MPASLIALB",
-                           "SMS_D_Ln5.conusx4v1_conusx4v1.FC5AV1C-L",
-			   ("SMS.ne30_oECv3.BGCEXP_BCRC_CNPRDCTC_1850","clm-bgcexp"),
-                           ("SMS.ne30_oECv3.BGCEXP_BCRC_CNPECACNT_1850","clm-bgcexp"))
-                          ),
-    #e3sm tests for extra coverage
-    "e3sm_extra_coverage" : (("e3sm_atm_extra_coverage",), None,
-                             ("SMS_D_Ln5.enax4v1_enax4v1.FC5AV1C-L",
-                              "SMS_D_Ln5.twpx4v1_twpx4v1.FC5AV1C-L")),
-
-    #e3sm tests for hi-res
-    "e3sm_hi_res" : (("e3sm_atm_hi_res",),None,
-                     (
-                      ("SMS.ne120_oRRS18v3_ICG.A_WCYCL2000_H01AS", "cam-cosplite"),
-                       "SMS.T62_oRRS30to10v3wLI.GMPAS-IAF",
-                     )),
-
-    #e3sm tests for RRM grids
-    "e3sm_rrm" : (None, None, 
-                  ("SMS_D_Ln5.conusx4v1_conusx4v1.FC5AV1C-L",
-                   "SMS_D_Ln5.enax4v1_enax4v1.FC5AV1C-L",
-                   "SMS_D_Ln5.twpx4v1_twpx4v1.FC5AV1C-L")
-                 ),
-
-    #e3sm tests to mimic production runs
-    "e3sm_prod" : (("e3sm_atm_prod",),None,
-                     (
-		      ("SMS_Ld2.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6","v1cmip6"),
-		      )),
-
-    "fates" : (None, None,
-                         ("ERS_Ld9.1x1_brazil.ICLM45ED",
-                          "ERS_D_Ld9.1x1_brazil.ICLM45ED",
-                          "SMS_D_Lm6.1x1_brazil.ICLM45ED")
-                         ),
-}
+import six, sys, os
+
+sys.path.append(os.path.join(get_cime_root(), "config", "e3sm"))
```

Comment:
In this line, can "e3sm" be replaced with $model?

Reply:
No, the cime test suites are needed for both models and cesm does not define their tests this way.